docs/quick-start/binding.md

```diff
@@ -118,6 +124,27 @@ The procedure is as follows:
     ![CONNECTED](../assets/images/LED_ON.gif)
     </figure>
 
+## Updated Binding Procedure since ExpressLRS 3.4.0
+
+With [PR#2542](https://github.com/ExpressLRS/ExpressLRS/pull/2542) being part of the 3.4.0 update, users now have the following methods to bind an ExpressLRS receiver with a TX module:
+
+1. Binding Phrase, flashed or set (via the WebUI) into the devices.
+2. 3x Power Cycle, Traditional Binding.
+3. Bind Receiver button in Betaflight Configurator 10.10 Receiver Tab (`bind_rx` CLI command also works). This will ONLY work if your flight controller is flashed with Betaflight 4.5.0.
+4. 1.5s Button Press on the Receiver itself.
+
+With the last 3 methods, the Receiver LED will blink twice, then cycles, until the Bind is established indicated by a solid LED light.
+
+<figure markdown>
+![LEDSEQ_BINDING](../assets/images/LEDSEQ_BINDING_10_10_10_100.gif)
+</figure>
+
+Receivers, however they are bound to a TX before, can be put into Bind Mode using any of the 3 methods above. This means that even if the Receiver has been flashed or set with a Binding Phrase, users can put them into Binding Mode any time. The Bind will persist or stay with the receiver until another Binding Procedure is done, with any of the methods above.
```

Comment:
`they are bound to a TX before`
seems like it should be 
`that are bound to a TX before`

Reply:
Thanks Wez,

I have reworded this section.